src/System.Runtime.Serialization.Xml/tests/SerializationTypes.cs

```diff
@@ -3157,6 +3157,12 @@ public class TypeWithTimeSpanProperty
     public TimeSpan TimeSpanProperty;
 }
 
+public class TypeWithDefaultTimeSpanProperty
+{
+    [DefaultValue(typeof(TimeSpan), "00:01:00")]
+    public TimeSpan TimeSpanProperty;
```

Comment:
Should there be a second TimeSpan property somewhere (here, or a different class) which has a different default, to show that the default is read from the property and not hard-coded?